### PR TITLE
BS4ify patient changelog and call log

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

Pretty straightforward changes to convert the call log and changelog partials to BS4. You can limit review to just `_call_log` and `_change_log` and `_change_log_entry`.

This pull request makes the following changes:
* BS4ify changelog
* BS4ify call log

olde:

image

image

new:

image

image

It relates to the following issue #s: 
* Bumps #1632

### CODE REVIEW

File: app/assets/stylesheets/pregnancies.scss [hunk 1]
```diff
@@ -21,52 +21,30 @@ $font-size: 17px;
...
-}
-
-.abortion-left-menu {
+.abortion-left-menu.nav-pills {
   font-size: 22px;
 
-  li {
-    // padding-left: 10px;
+  .nav-link {
+    color: #337ab7;
```
Comment (tingaloo) [inline app/assets/stylesheets/pregnancies.scss]: use daria-color

File: app/javascript/src/clinic_finder.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 $(document).on('turbolinks:load', () => {
   // Click on the clinic-locator expand to show it.
   $(document).on('click', '.clinic-finder-expand', () => {
-    $('#clinic-finder-search-form').toggleClass('hide');
+    $('#clinic-finder-search-form').toggleClass('hidden');
```
Comment (tingaloo) [inline app/javascript/src/clinic_finder.js]: d-none

File: app/views/clinicfinders/_search_form.html.erb [hunk 3]
```diff
@@ -1,46 +1,47 @@
...
+  <div class="card-header">
+    <h4 class="clinic-finder-expand patient-section-title">
+      <%= t('clinic_locator.title') %>
+      <small><%= t('clinic_locator.experimental_note') %></small>
+      <span class='glyphicon glyphicon-plus-sign float-right'>glyph</span>
+    </h4>
+  </div>
 
-  <div class='col-sm-12 hide' id='clinic-finder-search-form'>
+  <div class="card-body hidden" id="clinic-finder-search-form">
```
Comment (tingaloo) [inline app/views/clinicfinders/_search_form.html.erb]: d-none

File: app/views/notes/_notes_table.html.erb [hunk 4]
```diff
@@ -1,11 +1,13 @@
-<table class="table border-side" id='notes_log'>
+<table class="table border-side top-spacer" id="notes_log">
```
Comment (tingaloo) [inline app/views/notes/_notes_table.html.erb]: spacing util

File: app/views/patients/_abortion_information.html.erb [hunk 5]
```diff
@@ -1,110 +1,103 @@
...
-        <% end %>
+  <%= bootstrap_form_with model: patient, html: { id: 'abortion-information-form-2' }, remote: true do |f| %>
+    <div class="row patient-cost-fields">
+      <div class="col">
+        <%= f.number_field :procedure_cost,
+                           label: t('patient.abortion_information.cost_section.abortion_cost'),
+                           autocomplete: 'off',
+                           prepend: '$' %>
+
+        <div class="info-form-left form-group outstanding-balance-ctn hidden">
```
Comment (tingaloo) [inline app/views/patients/_abortion_information.html.erb]: d-none

File: app/views/patients/_patient_information.html.erb [hunk 6]
```diff
@@ -1,119 +1,149 @@
...
+                        label: t('patient.information.textable'),
+                        checked: patient.textable? %>
 
-          <%= f.select :voicemail_preference,
-                       options_for_select(voicemail_options,
-                                          patient.voicemail_preference),
-                       label: t('patient.information.voicemail_preference') %>
-          <div class="form-group">
-            <%= f.check_box :textable, label: t('patient.information.textable'), checked: patient.textable?  %>
+        <div class="row top-spacer">
```
Comment (tingaloo) [inline app/views/patients/_patient_information.html.erb]: use space util

File: app/views/patients/edit.html.erb [hunk 7]
```diff
@@ -5,13 +5,11 @@
...
-  <div>
-    <div id="menu" class="col-sm-3">
-      <%= render 'patients/menu', patient: @patient %>
-    </div>
+  <div id="menu" class="col-3">
+    <%= render 'patients/menu', patient: @patient %>
   </div>
 
-  <div id="sections" class="col-sm-9 tab-content">
+  <div id="sections" class="col tab-content">
```
Comment (tingaloo) [inline app/views/patients/edit.html.erb]: interesting name for an id name, should be a class?

Comment (colinxfleming) [reply app/views/patients/edit.html.erb]: probably true! I'll leave a todo

**Pull request merged**